### ISSUE TITLE
Polyglot tries to add existing nodes on startup.

### ISSUE THREAD

**Issue by jimboca** [UniversalDevicesInc/Polyglot#4]
Polyglot should check if a Node already exists before trying to add it?  I get these errors every time I restart.

WARNING  [03-11-2016 18:09:00] polyglot.element_manager.isy: Bad ISY Request: ht                                                                      tp://192.168.1.64:80/rest/ns/3/nodes/n003_f00626e495be9/add/FCAM1?primary=n003_f                                                                      00626e495be9&name=CameraGarage

**Comment by mjwesterhof** [comment]
Polyglot DOES check, and therein lies the problem -- according to a discussion on the forum quite some time ago, the issue is that the ISY end of the conversation is not completely implemented, so that Polyglot is not told that the node already exists.  Because of that, you get the above errors whenever Polyglot starts.

(Based on some scribbled notes I made as I investigated some other interfaces, I think there are a few others that may not be entirely implemented on the ISY side).

So that's where I'd start investigating this, were I you -- check to see if the ISY is correctly handling the API with 5.0.3 or if there's some tweaking required to get the add properly handled.
